torch/distributed/_tensor/api.py

```diff
@@ -523,8 +523,10 @@ def distribute_tensor(
     placements: Optional[Sequence[Placement]] = None,
 ) -> DTensor:
     """
-    Distribute a torch.Tensor to the `device_mesh` according to the `placements`
-    specified. The rank of `device_mesh` and `placements` must be the same.
+    Distribute a leaf torch.Tensor (i.e. nn.Parameter) to the `device_mesh` according
```

Comment:
nit: might need to check these renders later to make sure code type is used

Reply:
I actually haven't add the docs page yet in the pytorch documentation, plan to add a dedicated page in the follow up PRs and check the rendering